appside/model/dbt_worksheet_model.js

```diff
@@ -66,6 +69,24 @@ class DbtWorksheetModelFwd extends Model {
         ]);
     }
 
+    /**
+     * Build consent disclosure frames for a DBT worksheet model.
+     */
+    build_consent_disclosure_frames() {
```

Comment:
suggest renaming to `build_consent_disclosure_frame` since it only builds one.

Reply:
fixed